g3w-admin/base/settings/__init__.py

```diff
@@ -34,3 +35,10 @@
         from .tests_settings import *
     except ImportError:
         pass
+
+# Read SECRET_KEY from SECRET_KEY_FILE environment variable
+try:
+    with open(os.getenv('SECRET_KEY_FILE', '/shared-volume/.secret_key')) as f:
```

Comment:
Just to know, have you also considered this other topic? [Manage sensitive data with Docker secrets](https://docs.docker.com/engine/swarm/secrets/#use-secrets-in-compose) (related to: https://github.com/g3w-suite/g3w-suite-docker/issues/71)

If I'm not mistaken, within the compose file, it's possible to use the `secrets:` attribute without enable a "swarm" mode.

Reply:
@Raruto I think we can merge it, do you approve it?